yarn-project/end-to-end/src/composed/e2e_persistence.test.ts

```diff
@@ -342,23 +330,29 @@ describe('Aztec persistence', () => {
   });
 });
 
+function toBoundedVec(arr: Fr[], maxLen: number) {
```

Comment:
I think I've found this function in two or three other places in the codebase. Where should it live?

Reply:
Yeah I know. We don't always do it the same way though, returning a boundedvec from an oracle is not the same as calling a function with a bounded vec param. Given these are short snippets I thought it best to just add them as we go until we use them enough to warrant reuse.